spec/models/step_by_step_page_spec.rb

```diff
@@ -306,7 +306,7 @@
   end
 
   describe '#can_be_published?' do
-    let(:step_by_step_page) { create(:step_by_step_page_with_steps) }
+    let(:step_by_step_page) { create(:step_by_step_page_with_steps_but_no_draft) }
```

Comment:
Why has this changed to "no draft"?

Reply:
Should this be using `published_step_by_step_page` instead so that the tests still make sense?